rest-service/manager_rest/rest/resources_v1/executions.py

```diff
@@ -121,6 +123,16 @@ def post(self, **kwargs):
             scheduled_time=scheduled_time)
         return execution, 201
 
+    def _parse_scheduled_time(self, scheduled_time):
```

Comment:
Shouldn't this be in a separate PR?

Reply:
I needed it for the syncthing check in GET /status

Reply:
Fair enough, I'll mark this resolved then.